src/Aquifer.API/Endpoints/Notifications/BulkUpdate/Endpoint.cs

```diff
@@ -0,0 +1,118 @@
+﻿using Aquifer.API.Services;
+using Aquifer.Data;
+using Aquifer.Data.Entities;
+using FastEndpoints;
+using Microsoft.EntityFrameworkCore;
+
+namespace Aquifer.API.Endpoints.Notifications.BulkUpdate;
+
+public class Endpoint(AquiferDbContext _dbContext, IUserService _userService) : Endpoint<Request>
+{
+    public override void Configure()
+    {
+        Post("/notifications/update");
```

Comment:
@jwinston-bn, I used `update` in the route definition but named the folder `BulkUpdate` because the `Update` folder was already taken by the existing single notification PATCH route.  Does that naming work for you?